lib/stripe/callbacks.rb

```diff
@@ -65,12 +74,16 @@ module Callbacks
     callback 'recipient.created'
     callback 'recipient.deleted'
     callback 'recipient.updated'
+    callback 'review.closed'
+    callback 'review.opened'
+    callback 'scheduled_query_run.created'
```

Comment:
I'm not seeing this at the docs. Can you remove it please? Thanks!

Reply:
This should be `sigma.scheduled_query_run.created`